packages/vm/tests/api/runTx.spec.ts

```diff
@@ -230,8 +230,6 @@ tape('runTx() -> API parameter usage/data errors', (t) => {
       const tx = getTransaction(vm._common, txType.type, true)
       try {
         await vm.runTx({ tx })
-        // TODO uncomment:
-        // t.fail('should throw error')
```

Comment:
This should be in the test, right? If this `fail` is not reached then the test could pass if the `runTx` does not throw.

Reply:
Yes, it is in the tests added some commits later. If you comment on commits please always make sure to have a look at the final PR state, otherwise one is often falling in the trap (like here) of commenting on outdated code parts.